odl/discr/grid.py

```diff
@@ -72,34 +73,19 @@ def sparse_meshgrid(*x):
     return tuple(mesh)
 
 
-class TensorGrid(Set):
+class RectGrid(Set):
 
-    """An n-dimensional tensor grid.
+    """An n-dimensional rectilinear grid.
 
-    A tensor grid is the set of points defined by all possible
+    A rectilinear grid is the set of points defined by all possible
     combination of coordinates taken from fixed coordinate vectors.
 
-    In 2 dimensions, for example, given two coordinate vectors::
-
-        coord_vec1 = [0, 1]
-        coord_vec2 = [-1, 0, 2]
-
-    the corresponding tensor grid is the set of all 2d points whose
-    first component is from ``coord_vec1`` and the second component from
-    ``coord_vec2``. The total number of such points is 2 * 3 = 6::
-
-        points = [[0, -1], [0, 0], [0, 2],
-                  [1, -1], [1, 0], [1, 2]]
-
-    Note that this is the standard 'C' ordering where the first axis
-    (``coord_vec1``) varies slowest. Ordering is only relevant when
-    the point array is actually created; the grid itself is independent
-    of this ordering.
-
-    The storage need for a tensor grid is only the sum of the lengths
+    The storage need for a rectilinear grid is only the sum of the lengths
     of the coordinate vectors, while the total number of points is
-    the product of these lengths. This class makes use of this
-    sparse storage.
+    the product of these lengths. This class makes use of that
+    sparse storage scheme.
+
+    See ``Notes`` for details.
```

Comment:
Still not convinced that we should change the logical structure of the doc in order to please a specific IDE typesetting, other IDEs may do this "properly" (i.e. distinguish between a class and its initializer in a better way than spyder). Very few will read this anyway.